docs/proposals/compute-resource-metrics-api.md

```diff
@@ -80,16 +136,67 @@ namespace or service aggregation, metrics at those levels could also be
 exposed taking advantage of the fact that Heapster already does aggregation
 and metrics for them.
 
-Initially, this proposal included raw metrics alongside the derived metrics.
-After revising the use cases, it was clear that raw metrics could be left out
-of this proposal. They can be dealt with in a separate proposal, exposing them
-in the Kubelet API via proper versioned endpoints for Heapster to poll
-periodically.
+## Schema
+
+Types are colocated with other API groups in `/pkg/apis/metrics`, and follow api
+groups conventions there.
 
-This also means that the amount of data pushed by each Kubelet to the API
-server will be much smaller.
+```go
+type Node struct {
+  TypeMeta
+  ObjectMeta              // Should include node name
+  Machine ContainerMetrics
+  SystemContainers []ContainerMetrics
+}
+type Pod struct {
+  TypeMeta
+  ObjectMeta              // Should include pod name
+  Containers []Container
+}
+type Container struct {
+  TypeMeta
+  ObjectMeta              // Should include container name
+  Spec ContainerSpec      // Mirrors cadvisorv2.ContainerSpec
+  Stats []ContainerStats  // Mirrors cadvisorv2.ContainerStats
```

Comment:
I was trying to avoid copying the entire cadvisor API here. Is it useful to add one more level of depth here? For now I'll just add a link to the cadvisor API.